### PR TITLE
Button w/ circles

### PR DESCRIPTION
**Pull request by eliascodes**
**MERGE #323 FIRST**

Adds button component with the circles in the corners, CSS copied from the twine website, but translated to styled-components.

screen shot 2018-04-01 at 17 56 38

### CODE REVIEW

File: react-backend/client/src/shared/components/form/DottedButton.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  transform: translate3d(-50%, 50%, 0);
+`;
+
+const DotBR = Dot.extend`
+  bottom: 0;
+  right: 0;
+  transform: translate3d(50%, 50%, 0);
+`;
+
+const DotButton = ({ children, ...props }) =>
```
Comment (astroash) [inline react-backend/client/src/shared/components/form/DottedButton.js]: CSS 👑

File: react-backend/client/src/cb-admin/pages/Dashboard.js [hunk 2]
```diff
@@ -17,7 +18,7 @@ const FlexItem = styled.div`
   flex: ${props => props.flex || '1'};
 `;
 
-const ButtonOne = PrimaryButtonNoFill.extend`
```
Comment (astroash) [inline react-backend/client/src/cb-admin/pages/Dashboard.js]: Should this be `Component.extend`?

File: react-backend/client/src/shared/pages/Home.js [hunk 3]
```diff
@@ -26,14 +27,14 @@ const FlexItem = styled.div`
   flex: ${props => props.flex || '1'};
 `;
 
-const ButtonLeft = styled(PrimaryButtonNoFill)`
-  width: 30vh;
-  height: 25vh;
+const ButtonLeft = styled(DotButton)`
```
Comment (astroash) [inline react-backend/client/src/shared/pages/Home.js]: Same here, should it be extend?

File: react-backend/client/src/cb-admin/pages/Dashboard.js (hunk 2 above)
Comment (eliascodes) [reply react-backend/client/src/cb-admin/pages/Dashboard.js]: Maybs...will try.

File: react-backend/client/src/shared/components/form/DottedButton.js (hunk 1 above)
Comment (eliascodes) [reply react-backend/client/src/shared/components/form/DottedButton.js]: 🤫

**Comment by eliascodes** [comment]
Please use `rebase and merge` option from the drop-down when merging.

**Comment by astroash** [comment]
@eliascodes can you resolve linter errors and rebase?

File: react-backend/client/src/cb-admin/pages/Dashboard.js (hunk 2 above)
Comment (eliascodes) [reply react-backend/client/src/cb-admin/pages/Dashboard.js]: This causes a runtime error, so wrapping is the correct way.

File: react-backend/client/src/shared/pages/Home.js (hunk 3 above)
Comment (eliascodes) [reply react-backend/client/src/shared/pages/Home.js]: This causes a runtime error, so wrapping is the correct way.

**Comment by eliascodes** [comment]
@RogeredBacon ditto with this one, if we could merge b/c @astroash has already seen it.

**Pull request merged**